src/Traits/ReflectionClassLikeTrait.php

```diff
@@ -698,8 +699,16 @@ public function isInstance($object)
         }
 
         $className = $this->getName();
-
-        return $className === get_class($object) || is_subclass_of($object, $className);
+        $objClass = get_class($object);
```

Comment:
Please revert because Scrutinizer CI report (see https://scrutinizer-ci.com/g/goaop/parser-reflection/inspections/eaf283f6-f982-44a4-bb28-51489bcba537/issues/files/src/Traits/ReflectionClassLikeTrait.php?status=fixed&orderField=path&order=asc&honorSelectedPaths=0) mentions https://bugs.php.net/bug.php?id=53727 bug, which was fixed in PHP 5.3 and PHP 5.4, but this library has PHP 5.5+ as minimum required version.

Therefore bug never manifested itself in here.